src/Universum.hs

```diff
@@ -214,7 +215,7 @@ readEither :: (ToString a, Read b) => a -> Either Text b
 readEither = X.first toText . Text.Read.readEither . X.toString
 
 -- | Generalized version of 'Prelude.show'.
-show :: (Show a, IsString b) => a -> b
+show :: forall b a . (Show a, IsString b) => a -> b
```

Comment:
I would like you to perform a little refactoring: move `String`-related functions and types and reexports from `Universum` module to `Conv` module. And then rename `Conv` module to `String`. The idea is to have in `Universum` only reexports of other `universum` modules. This refactoring is performed lazily.. But I would like you to do this for `String`-related functionality.